apps/projects/templates/meinberlin_projects/includes/project_list_tile.html

```diff
@@ -3,8 +3,9 @@
 <div class="tile">
     <div class="tile__head">
         <a tabindex="-1" class="tile__image tile__image--shadowed"
+           {% if project.image %}style="background-image: url({{ project.image|thumbnail_url:'project_thumbnail' }})"{% endif %}
            href="{% extproject_url project 'project-detail' %}" target="{{ project|is_external|yesno:"_blank,_self" }}">
-            <img src="http://placehold.it/285x170" alt="">
+            <span class="sr-only">{{ project.name }}</span>
```

Comment:
`project.name` is already mentioned in `tile__title`. I would add [`role="presentation"`](https://www.w3.org/TR/wai-aria/roles#presentation) to the whole link instead.